src/main/webapp/search-style.css

```diff
@@ -12,31 +12,30 @@ label, select {
     width: 120px;
 }
 
-#search-textarea, #search-selectboxes, #class-textarea, #class-selectboxes {
-    display: flex;
+#search-button:hover, #logout-button:hover {
+  background-color: #45a049;
 }
 
-textarea, select, fieldset {
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  box-sizing: border-box;
-  display: inline-block;
-  margin: 8px 0;
-  padding: 12px 20px;
-  width: 50%;
+#private-class-info {
+  margin: auto;
+  width: 80%;
 }
 
-.course-button {
-  background-color: #4CAF50;
-  border: none;
-  border-radius: 4px;
-  color: white;
-  cursor: pointer; 
-  margin: 8px 0;
-  padding: 14px 20px;
-  width: 100%;
+.button-container {
+  margin: auto;
 }
 
-#search-button:hover, #logout-button:hover {
-  background-color: #45a049;
+#private-logout-button {
+  position: absolute;
```

Comment:
Super nit: Alphabetize these

Reply:
Oops, I think I alphabetized this in a later branch. Made this change on this branch as well.